src/fragmentarium/ui/display/Transliteration.tsx

```diff
@@ -109,32 +112,89 @@ class LineAccumulator {
     )
   }
 }
+function DisplayLineNumberRange({ start, end }: LineNumberRange): JSX.Element {
```

Comment:
It is probably not a good idea to use a model type as prop type. Better to pass the line number as a single prop. You can still use destructuring e.g. `{range: { start, end}}: {range: LineNumberRange}`.